manifests/init.pp

```diff
@@ -18,6 +18,25 @@
   $service_status     = $xinetd::params::service_status,
   $service_hasrestart = $xinetd::params::service_hasrestart,
   $service_hasstatus  = $xinetd::params::service_hasstatus,
+  $enabled            = false,
+  $disabled           = false,
+  $log_type           = false,
+  $log_on_failure     = false,
+  $log_on_success     = false,
+  $no_access          = false,
+  $only_from          = false,
+  $max_load           = false,
+  $instances          = false,
+  $per_source         = false,
+  $bind               = false,
+  $mdns               = false,
+  $v6only             = false,
+  $passenv            = false,
+  $groups             = false,
+  $umask              = false,
+  $banner             = false,
+  $banner_fail        = false,
+  $banner_success     = false,
```

Comment:
Since these are supposed to accept strings, not bools, could you default them to `undef` instead of `false` ?